contrib/builder/rpm/ppc64le/README.md

```diff
@@ -0,0 +1,5 @@
+# `dockercore/builder-rpm`
```

Comment:
You can also drop this README. Amd64 is the only other architecture to have one in all of the deb/rpms, and IMO it's not really needed or useful. Keeps consistent with s390x in this PR too :)